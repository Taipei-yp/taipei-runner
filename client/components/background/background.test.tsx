import React from "react";
import TestRenderer from "react-test-renderer";
import { Background } from ".";

describe("Component Background", () => {
  const avatar = TestRenderer.create(<Background />);
  it("correct render", () => {
    expect(avatar.toJSON()).toMatchSnapshot();
  });
});
