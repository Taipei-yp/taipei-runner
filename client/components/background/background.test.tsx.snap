// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component Background correct render 1`] = `
<div
  className={
    Object {
      "has": [Function],
      "is": [Function],
      "mix": [Function],
      "split": [Function],
      "state": [Function],
      "toString": [Function],
    }
  }
/>
`;
